modyn/selector/internal/selector_strategies/downsampling_strategies/abstract_downsampling_strategy.py

```diff
@@ -57,6 +56,10 @@ def _compute_status_bar_scale(self) -> int:
             return 100
         return self.downsampling_ratio
 
+    @property
+    def downsampling_params(self) -> dict:
+        return self._build_downsampling_params()
```

Comment:
Actually should we get rid of `_build_downsampling_params()` and just let subclasses override `downsampling_params`? Not sure we need the extra level of indirection. Then subclasses can just use `@cached_property` and RHO LOSS uses @property

Reply:
good idea!